spec/support/vcr_cassettes/travel_pay/documents/get/internal_error.yml

```diff
@@ -2,7 +2,7 @@
 http_interactions:
 - request:
     method: get
-    uri: https://btsss.gov/api/v2/claims/claim-123/documents/big-bad-error
+    uri: https://btsss.gov/api/v2/claims/73611905-71bf-46ed-b1ec-e790593b8565/documents/big-bad-error
```

Comment:
Please add uri to [vcr support](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/support/vcr.rb) to mask the URI with a env variable.